### PR TITLE
fix typo in docs-contributor-technical-guide

### PR DESCRIPTION
**Pull request by pawanharariya**
addresses odk-x/tool-suite-X#207

#### What is included in this PR?
Fixed typo in the setup collaboration accounts section of docs contributor technical guide

#### What is left to be done in the addressed issue?
Why haven't we mentioned about **Slack** here, and in the section `should I ask in the forum or a github issue` there are two parts mentioning `on Github` and `on Github issue`.  Are they suppose to be different? I guess one of them should have been slack.

### CODE REVIEW

File: odkx-src/docs-tech-guide.rst [hunk 1]
```diff
@@ -139,7 +139,7 @@ and the :ref:`ODK-X Forum <join-forum>`
 
    It is better to ask a question in the "wrong" venue
    than to not ask the question at all.
-   Many of the same people are present in all three places,
+   Many of the same people are present in both the places,
```
Comment (elmps2018) [inline odkx-src/docs-tech-guide.rst:142]: minor edit: both places rather than both the places

**Review by elmps2018 — COMMENTED** [review]
Thanks for working to fix this!

**Comment by elmps2018** [comment]
For what should be on Github generally versus a Github issue I will ask @linl33 or @wbrunette to weigh in!

File: odkx-src/docs-tech-guide.rst (hunk 1 above)
Comment (pawanharariya) [reply odkx-src/docs-tech-guide.rst:142]: I first wrote both places and later changed it to both the places, really weak at this :'(
I will fix this.

**Comment by wbrunette** [comment]
It's GitHub. https://en.wikipedia.org/wiki/GitHub

**Review by wbrunette — APPROVED** [review]

**Pull request merged**